cmd/stack/deploy.go

```diff
@@ -64,9 +64,6 @@ func Deploy(ctx context.Context) *cobra.Command {
 			if err == nil {
 				log.Success("Stack '%s' successfully deployed", s.Name)
 			}
-			if err := stack.ListEndpoints(ctx, s, ""); err != nil {
```

Comment:
Why are we removing it from stacks? Are we going to start using the deploy command for stacks?

Reply:
If we deploy an stack through deploy command it will appear two times the endpoints available. I think that the deploy command will be the option to go as a long term command so removing it here and only showing it from the deploy command will make users run `okteto deplo`y instead of `okteto stack deploy`

Reply:
but we are penalizing the `okteto stack deploy` command for `okteto deploy` command which is still hidden for users.  Maybe we can make it optional. Show the endpoints for stacks but using an environment variable (I think it is too much for this case) or something similar to do not display them when executed through `okteto deploy`

Reply:
yes, let's do this only if we are inside a `deploy` command